packages/backend-plugin-api/api-report.md

```diff
@@ -136,6 +137,15 @@ export type BackstageNonePrincipal = {
   type: 'none';
 };
 
+// @public
+export type BackstagePrincipalAccessRestrictions = Array<{
+  pluginId: string;
```

Comment:
Hmm, not sure this should be here. At this point we've already validated the plugin right? I think it would even make sense to unpack the permission restrictions to apply to this plugin, i.e. this shouldn't be an array

Reply:
Especially if we aren't going to send this transitively to upstreams